addons/hr_attendance/models/hr_attendance.py

```diff
@@ -433,3 +436,33 @@ def action_out_attendance_maps(self):
             'url': get_google_maps_url(self.out_latitude, self.out_longitude),
             'target': 'new'
         }
+
+    def load_demo_data(self):
+        convert.convert_file(self.env, 'hr_attendance', 'data/scenarios/hr_attendance.xml', None, mode='init', kind='data')
+        date_range =  datetime.today().day
+        attendance_values = []
+        employee_sj = self.env.ref('hr_attendance.employee_sj')
+        attendance_values = [{
+            'employee_id': employee_sj.id,
+            'check_in': datetime.now().replace(hour=6, minute=0)+timedelta(days=-i, minutes=randint(-5,5)),
+            'check_out': datetime.now().replace(hour=15, minute=0)+timedelta(days=-i, minutes=randint(-5,5)),
+            'in_mode': "manual"
+        } for i in range(1, date_range)]
```

Comment:
If you load the scenario on the 1rst of the month, you won't create any attendance btw.